### ISSUE TITLE
Better documentation around named snippets

### ISSUE THREAD

**Issue by litghost** [antmicro/tuttest#12]
While debugging https://github.com/SymbiFlow/symbiflow-examples/pull/50, it is hard to follow the current strategy of naming tests by their order in a README.  It also strikes me as fragile.  

The README mentions a naming scheming, but doesn't provide a lot of context on how to use it.

Questions and requests:
 - How can snippets be named, so that invocation like https://github.com/SymbiFlow/symbiflow-examples/blob/master/.travis.yml#L28 can be clearer
 - Can we add some Travis tests to tuttest itself to both demostrating the naming features, and to insure they do not break?

**Comment by mgielda** [comment]
Sure! We'll get to it on Monday.

**Comment by mgielda** [comment]
In the meantime, IIRC it's the 'name' option in rst and a html comment in md that accomplishes this.

**Comment by mgielda** [comment]
@litghost hope this helps? The README describes it now.

**Comment by mgielda** [comment]
(please let us know if the explanation is enough for you and then we will close this)
